### ISSUE TITLE
Chris's Field Report 

### ISSUE THREAD

**Issue by c-johnson** [urbit/bridge#40]
tldr; I wrote down stream-of-consciousness thoughts as I used bridge for the first time, with no crypto experience & only marginal understanding of various Azimuth tickets/wallets/seeds/keys. 

Only big issue seem to be I can't seem to do anything with my star keys (which are maybe disabled on purpose?). Other than that, this process is incredibly smooth, and the rest of the notes are all relatively minor; really great work everyone.

===

Task 1: I have an urbit planet (~ridhec-salput) wallet-generator'd PDF that I want to instantiate a pier on my computer with to connect to the Urbit network. 

  - First page I see is "Create A Galaxy". 
    - I don't have a galaxy! 
    - I was able to click "Bridge" from the breadcrumb and start from there, though, so not too bad. 

  - "Select Network". 
    - I have no idea what these various nodes are. I just clicked "Main Network" but I feel uncomfortable about it. 

  - Mostly at this point, I'm worried that if I click any button, I'm creating Eternal Ramifications for my property for having not understood everything correctly. It seems like there's no actual ramifications until I select an "action" after my wallet is loaded, so maybe something to the effect of "feel free to click around, it's hard to hurt yourself until you click an Action"

  - Speaking of Eternal Ramifications; I'm connected to the internet when I load bridge; am I also "loading" my "private key" in any sense? Why does wallet-generator have to be offline, but bridge is fine online? 

  - I'm on "Open a Wallet" now. 
    - Again, I don't know what these options are. 
    - I notice that I have 2 pdfs generated from the wallet-generator process; on the PDF labeled "master ticket", I have both a "master ticket" and a "bip32 mneumonic".
    - At this point in the process, I know neither `which PDF` I want in order to `instantiate a ship`, nor `which property` in the PDF I want to select for this option. 
    - After playing around with it for a minute, it looks like which key you enter unlocks different actions when you reach the point detail page. So I figured it out eventually.

  - I'm on the "point detail" page for ~ridhec-salput
    - I want to "instantiate a pier on my computer", but the only option I see here is "Generate Urbit keyfile". I'm guessing this has to be the action. This gives me a private key. Presumably there's some urbit/urbit incantation that consumes this private key to give me a pier, if that's on new-urbit.org, that's fine.

Task 2: I have an urbit star (~nosdef) that I want to spawn a planet with (~unknown).
    - I'm assuming I use my "spawn seed":
      - After entering my spawn seed pneumonic, the "Points" page displays:
        Your Points
        no points to display 
      - If I click "view a point", and enter in ~nosdef, it occasionally seems to error out and redirect me to the "Create a galaxy" page 
      - After trying a couple of times, I can view the ~nosdef "point detail" page, but no actions are available. 
    - Trying again with my "master ticket":
      - I notice on my Master Ticket pdf that I have two @p master tickets:
        ~xxxxxx-xxxxxx-xxxxxx-xxxxxx
        ~xxxxxx-xxxxxx-xxxxxx-xxxxxx
        - I'm not sure which one to use, so I tried the first one:
          - Again, same problem with "spawn seed". The Your Points section prints "no points to display"
        - Trying again with the second one:
          - Again, same problem. The Your Points section prints "no points to display"
        - Trying again with bip32 mneumonic:
          - Again, same problem. The Your Points section prints "no points to display"
      - I'm thinking there might be an issue with having 2 bridge tabs open, or some state that hasn't been hooked up properly. So I'll restart the server and clear the browser cache.
      - Restarted server. Same issues above, no matter which key pneumonic/ticket I try.
    - Ah, I figured it out! I was on "Local Node" the whole time. 
      - Well, I'll leave the above as a stream-of-consciousness report when you click a wrong thing. Not sure if there's any action to take here.
    - Okay, so I loaded my "spawn seed" pneumonic, and every action is greyed out. 
      - "Spawn Proxy" matches up to "Spawn Address" in my PDF, so I believe this is a valid wallet.
      - I would have expected this to unblock the "Issue Child" action. 
        - It does say "Points you can spawn on behalf of" on the "points index" page.
    - Trying again with "master ticket" mneumonic.
      - Seems to log me in, but I can't issue children, only set other keys. Makes sense. 
    - Trying again with "management seed" mneumonic.
      - Only gives me "Set networking keys" action. 
    - Seems overall like I can't figure out how to take any action on my star except "Set networking keys", and "Set <blank> proxy" w/ master ticket. Giving up for now.

===
